src/Function/Properties/Bijection.agda

```diff
@@ -0,0 +1,52 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Conversions for bijections
+--   This file is meant to be imported qualified.
```

Comment:
Why is this module supposed to be imported qualified out of interest?

Reply:
No reason, I just copied some stuff around and accidentally left that there